flink-catalog-examples/docker-compose.yml

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright Pravega Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+version: '2.1'
+services:
+  sql-client:
+    image: fyang86/catalog-demo-sql-client:0.1  # change to your own docker repo if you want to build this image
```

Comment:
In general, I don't like setting personal repos in code. Can we just change this by something else that may help the user to know that has to actually build the image? The alternative is to pushing it to oficial Pravega dockerhub.

Reply:
Agree, I'll this image to official Pravega dockerhub